src/ceph_mon.cc

```diff
@@ -298,25 +298,25 @@ int main(int argc, const char **argv)
     int err = check_mon_data_exists();
     if (err == -ENOENT) {
       if (::mkdir(g_conf->mon_data.c_str(), 0755)) {
-	cerr << "mkdir(" << g_conf->mon_data << ") : "
-	     << cpp_strerror(errno) << std::endl;
+	derr << "mkdir(" << g_conf->mon_data << ") : "
```

Comment:
This would be nice to have, but I wonder whether `cerr` is actually more appropriate.